job-wrappers/default_singularity_wrapper.sh

```diff
@@ -143,7 +143,10 @@ if [[ $GWMS_SINGULARITY_REEXEC -ne 1 ]]; then
     if [ -e "$group_dir/itb-ospool-lib" ]; then
         source "$group_dir/itb-ospool-lib" || exit_wrapper "Unable to source itb-ospool-lib" 1
     else
-        source "$group_dir/ospool-lib" || exit_wrapper "Unable to source ospool-lib" 1
+        source "$group_dir/ospool-lib" || {
+            error_message="Unable to source ospool-lib; group_dir is $group_dir; $(ls -ld "$group_dir"); $(ls -ld "$group_dir/ospool-lib")"
+            exit_wrapper "$error_message" 1
+        }
```

Comment:
I just spoke with Greg on this one: this error message currently goes into the Shadow/Starter logs, which cannot handle multi-line messages. He suggests dumping to stderr, which we believe will get dumped into the job's stderr

Reply:
It's not multiline, I `ls` one file/directory at a time and separate the results with a ';'

Reply:
Ah, good call

Reply:
The problem with dumping into stderr is then we have to track down which job has the error and find its stderr logs